### PR TITLE
This resolves #4640

### LINKED ISSUE #4640: Update basic.md `generic installation`.
**I'm submitting a ...**  (check one with "x")
- [x] documentation update
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-server

**Current behavior:**
[local installation](https://github.com/fossasia/open-event-server/blob/development/docs/installation/local.md) vs [generic installation](https://github.com/fossasia/open-event-server/blob/development/docs/installation/local.md)

In local installation, instruction to copy the .env.sample file is given which is not present in the basic installation. 

And also in local installation, postgres user john is created.
>CREATE USER john WITH PASSWORD 'start';
CREATE DATABASE oevent WITH OWNER john;

which works fine when running `python create_db.py` as in the `.env.sample` file `DATABASE_URL` is-

`DATABASE_URL=postgresql://john:start@127.0.0.1:5432/oevent `

whereas in basic installation -
>create user open_event_user with password 'test';
create database test with owner=open_event_user;

so when running `python create_db.py`, it will result in error as it is also using the DATABASE_URL and no oevent database and no john user is there in this case.

These two will be issues for the newcomers who might try `generic installation` instead of `local installation` since it is the first option in installation section.

### PR DESCRIPTION
**Pull request by harsh-98**
Fixes #4640

### CODE REVIEW

File: docs/installation/basic.md [hunk 1]
```diff
@@ -49,22 +49,33 @@ sudo -u postgres psql
 
 * When inside psql, create a user for open-event and then using the user create the database.
 
+For ease development, you should create Postgres user with the same username as your OS account. If your OS login account is _john_, for example, you should create _john_ user in Postgres. By this, you can skip entering password when using database.
```
Comment (shubham-padia) [inline docs/installation/basic.md]: This suggestion does not seem necessary to me, different users are created for a database to manage access to multiple databases. Although, users are free to use their choice of user, we cannot suggest them this :)

File: docs/installation/basic.md [hunk 2]
```diff
@@ -49,22 +49,33 @@ sudo -u postgres psql
...
 
 
-* **Step 3** - Start the postgres service.
+* **Step 3** - Create application environment variables.
+
+```sh
+cp .env.example .env
+```
+
+The URL is short, thanks to the resemble of Postgres user and OS user.
```
Comment (shubham-padia) [inline docs/installation/basic.md]: Making the URL short is not a concern

**Comment by shubham-padia** [comment]
@harsh-98 Thank you for your contribution.

I've left some comments requesting changes. Also please rename the commit to something appropriate like `Installation docs: Use same postgres user thorughout the guide.`
Please look at the past commits by the core contributers to get a idea of the commit style and also please have a look at our best practices in the README :)

**Comment by harsh-98** [comment]
done :+1:

**Comment by shubham-padia** [comment]
Please squash both the commits into one and give an appropriate commit message :)

**Comment by harsh-98** [comment]
done :+1:

**Comment by harsh-98** [comment]
@shubham-padia i have done the requested changes

**Comment by harsh-98** [comment]
@shubham-padia are these changes fine to be merged?

**Comment by iamareebjamal** [comment]
@harsh-98 A suggestion, please make the commit messages a bit more descriptive as they may be the only content a developer has on a local machine when viewing a log or quickly going through recent changes

**Comment by harsh-98** [comment]
@iamareebjamal done

**Pull request merged**